frontend/lib/src/components/widgets/FileUploader/FileUploader.tsx

```diff
@@ -167,6 +167,24 @@ class FileUploader extends React.PureComponent<Props, State> {
     }
   }
 
+  public componentDidMount(): void {
+    const newWidgetValue = this.createWidgetValue()
+    if (newWidgetValue === undefined) {
+      return
+    }
```

Comment:
nit (also applies to some places in this file that don't show up in the diff):

Looking at `createWidgetValue`, I don't think it's possible for the method to return anything falsy anymore. This must have been something that changed with the file uploader abstractions changes and we missed. We can probably get rid of this guarding if block + others like it